e2e/updatecli.d/success.d/gitlab/branch.yaml

```diff
@@ -1,4 +1,4 @@
-name: "Test Gitlab branch"
+name: "Test GitLab branch"
```

Comment:
Brand (I don't think there are any other brands being corrected in this PR, and I wouldn't generally flag more than 2)